src/sqs/queues.ts

```diff
@@ -56,7 +56,7 @@ const sqsQueues = {
 		maxAttempts: 5
 	},
 	deploymentQueueMessageHandler,
-	webhookMetricWrapper(jiraOctokitErrorHandler, "deployment")
+	webhookMetricWrapper(jiraOctokitErrorHandler, "deployment_status")
```

Comment:
That was an issue from a previous PR with the deployment queue. It would count the failing events against the wrong metric name 😬